incubator/quarkus/templates/default/.appsody-config.yaml

```diff
@@ -0,0 +1,2 @@
+# config file for application development
+stack: pfrobinson/quarkus-appsody-stack:latest
```

Comment:
can you change this is point to the appsody repo, example:
`appsody/java-quarkus:0.1`

Reply:
Is that better?